src/admin/row/id.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { CenterCell } from 'admin/components/cell/center-cell'
+
+export const Id = ({ user }) => <CenterCell component={'th'} scope={'row'} align={'center'}>
```

Comment:
These props are not applied. Let's remove them.

Reply:
done